test/parallel/test-resources-usage.js

```diff
@@ -0,0 +1,12 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
You can remove the copyright entirely!

Reply:
My bad i wasn't scrolling to the top of the file ahah

Reply:
I've unresolved this conversation because a copyright line is still present.